src/app/util/ContentApp.cpp

```diff
@@ -46,20 +46,43 @@ namespace AppPlatform {
 EmberAfStatus ContentApp::HandleReadAttribute(ClusterId clusterId, chip::AttributeId attributeId, uint8_t * buffer,
                                               uint16_t maxReadLength)
 {
-    ChipLogProgress(DeviceLayer, "Read Attribute for device %s cluster %d attribute=%d)",
+    ChipLogProgress(DeviceLayer, "Read Attribute for device %s cluster %d attribute=%d",
                     GetApplicationBasic()->GetApplicationName(), static_cast<uint16_t>(clusterId),
                     static_cast<uint16_t>(attributeId));
 
-    EmberAfStatus ret = EMBER_ZCL_STATUS_FAILURE;
     if (clusterId == ZCL_APPLICATION_BASIC_CLUSTER_ID)
```

Comment:
Please use the namespaced cluster id constants, not the ones we are trying to remove.

Reply:
this is a moving target. Is there info somewhere you can point me to which explains what constants you are trying to remove and which ones we should be using?

Reply:
We want `app::Clusters::ApplicationBasic::Id` instead of `ZCL_APPLICATION_BASIC_CLUSTER_ID`.

Reply:
the plan is to remove all of this logic once Lazar has finished refactoring the media cluster code into delegate instances similar to what is done with content launcher here.